python-avd/pyavd/_eos_designs/structured_config/network_services/port_channel_interfaces.py

```diff
@@ -52,6 +168,14 @@ def port_channel_interfaces(self: AvdStructuredConfigNetworkServicesProtocol) ->
                 if potential_parent_interface.name not in configured_physical_po:
                     self.structured_config.port_channel_interfaces.append(potential_parent_interface)
 
+        # Sanity check if there are any sub-interfaces for which parent Port-channel is not explicitly specified
```

Comment:
this is true only for l3_port_channels right not for point to point?

Reply:
correct. It could be moved above the point to point if needed.